tensorflow/contrib/verbs/rdma.cc

```diff
@@ -1086,7 +1086,7 @@ void RdmaTensorResponse::RecvHandler(Rendezvous::ParsedKey parsed,
       // The tensor must be copied from GPU to CPU, because either:
       // 1. The tensor is located on a non GDR compatible GPU.
       // 2. The tensor's meta-data has changed.
-      Allocator* alloc = GPUProcessState::singleton()->GetCUDAHostAllocator(0);
+      Allocator* alloc = GPUProcessState::singleton()->GetGPUHostAllocator(0);
```

Comment:
I'm aware that we are far from consistent, but could you please change the 'GPU's you introduce to 'Gpu'?

Reply:
done.
I changed all `GPUHostAllocator` instances to `GpuHostAllocator`. Although as you have already mentioned, there remain a lot of references to `*GPU*` within the the code in `tensorflow/core/common_runtime/gpu`

Reply:
Thanks a lot. I will take care of the remaining 'GPU' names in a separate CL.